source/use-cases/sensitive-data-encryption.txt

```diff
@@ -300,6 +289,17 @@ information:
 - The BSON Type of each field (only required for deterministically
   encrypted fields)
 
+.. admonition::JSON Schema Validation
+   :important:
+   
+   MongoDB drivers only use JSON Schema to configure automatic client-side
+   field level encryption. If you specify a document validation instance 
+   without an ``encrypt`` property, the client will either ignore it or throw
```

Comment:
Maybe link to https://docs.mongodb.com/manual/reference/security-client-side-automatic-json-schema/ for supported keywords and explicitly state that any validation-specific keywords result in errors?

Reply:
Thanks for the link. I'll incorporate this into this admonition.